### PR TITLE
Update the font page. New content and font sample tool.

### LINKED ISSUE #29: Build /font
Build the changes to the font page

### PR DESCRIPTION
**Pull request by deadlight**
## Done

 - New content and layout for the /font page
 - Rewrote the font testing tool

## QA

 - `./run`
 - [Go to /font](http://localhost:8011/font/)
 - Test the font tool and check the page
  - [Design](https://www.dropbox.com/work/00_web_team/_design.ubuntu.com/Font%20page/Exports?preview=001+I+Fonts+page+-+final+designs.jpg)
  - [Copydoc](https://docs.google.com/document/d/1KeJYwl7Cu4kMYo2X3oiwmYn0hb_D-NTA0NcCAVAET_k/edit#)

## Issue / Card

Fixes #29 

## Demo
http://design.ubuntu.com-font.demo.haus/font/

### CODE REVIEW

File: font/index.md [hunk 1]
```diff
@@ -5,85 +5,99 @@ body-id: fonts
...
-      <a href="./about">More about the Ubuntu Font Family &rsaquo;</a>
+      <h2>About the Ubuntu font family</h2>
+      <p>The Ubuntu font family are a set of matching new libre/open fonts. The development is being funded by <a href="http://www.canonical.com/" class="p-link--external">Canonical</a> on behalf the wider Free Software community and the Ubuntu project. The technical font design work and implementation is being undertaken by <a href="http://www.daltonmaag.com/" class="p-link--external">Dalton&nbsp;Maag</a>.</p>
+      <p>Both the final font Truetype/OpenType files and the design files used to produce the font family are distributed under an <a href="http://font.ubuntu.com/licence" class="p-link--external">open licence</a> and you are expressly encouraged to experiment, modify, share and improve. The typeface is sans-serif, uses OpenType features and is manually hinted for clarity on desktop and mobile computing screens.</p>
```
Comment (anthonydillon) [inline font/index.md]: There is a link to font.ubuntu.com which this is due to replace. This link should be updated.

File: font/index.md [hunk 2]
```diff
@@ -5,85 +5,99 @@ body-id: fonts
...
+      </ul>
+    </div>
+  </div>
+</div>
+
+<div class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Resources</h2>
+      <p><a href="" class="p-button--brand">Download for free</a></p>
```
Comment (anthonydillon) [inline font/index.md]: Missing a href.

File: font/index.md [hunk 3]
```diff
@@ -5,85 +5,99 @@ body-id: fonts
...
+      <ul class="p-list--divided">
+        <li class="p-list__item js-font-test-input"><a href="https://assets.ubuntu.com/v1/fad7939b-ubuntu-font-family-0.83.zip" class="p-link--external">Download the Ubuntu Font Family</a> (2.0MB)</li>
```
Comment (anthonydillon) [inline font/index.md]: Remove js class here.

File: assets/js/font.js [hunk 4]
```diff
@@ -1,90 +1,36 @@
...
 
-
-// Get the current page ID
-base.getPageId = function() {
-  return $('body').attr('class');
-};
+document.addEventListener(
+  'DOMContentLoaded',
+  function (loadEvent) {
+    core.fontSelecter();
```
Comment (anthonydillon) [inline assets/js/font.js]: This function should be called from the footer of the page. Not within the module itself.

File: assets/js/font.js [hunk 5]
```diff
@@ -1,90 +1,36 @@
...
-        .children('option[value!=300][value!=500][value!=700][value!=italic]')
-        .attr('disabled', false);
-    } else if (value === 'Ubuntu') {
-      $('select')
-        .children('option')
-        .attr('disabled', false);
+  }
+
+  // Disable the options that are not available with current selected options.
+  function filterOptions(value){
```
Comment (anthonydillon) [inline assets/js/font.js]: Add a space between ) and {

File: assets/js/font.js [hunk 6]
```diff
@@ -1,90 +1,36 @@
...
+    options.forEach(function(option){
```
Comment (anthonydillon) [inline assets/js/font.js]: Add a space between ) and {

File: assets/js/font.js [hunk 7]
```diff
@@ -1,90 +1,36 @@
...
+      option.disabled = false;
+    });
+
+    //Disable light and medium font weight for condensed and monospace font faces
+    if (value === 'condensed' || value === 'monospace') {
+      options.forEach(function(option){
```
Comment (anthonydillon) [inline assets/js/font.js]: Add a space between ) and {

File: assets/js/font.js [hunk 8]
```diff
@@ -1,90 +1,36 @@
...
-        .attr('disabled', false);
+  }
+
+  // Disable the options that are not available with current selected options.
+  function filterOptions(value){
+    options.forEach(function(option){
+      option.disabled = false;
+    });
+
+    //Disable light and medium font weight for condensed and monospace font faces
```
Comment (anthonydillon) [inline assets/js/font.js]: Add space after //

**Comment by spencerbygraves** [comment]
- [ ] In Chrome the form styling is wrong on large screen. It seems fine on small screen

**Large**
image

**Small**
image

- [ ] 'Type your text here' does not match the design

Thanks

**Comment by anthonydillon** [comment]
@spencerbygraves the design failings are due to a lack of autoprefixer on the site. Which is another issue.

**Pull request merged**